src/pages/patientView/mutation/column/AnnotationColumnFormatter.tsx

```diff
@@ -0,0 +1,279 @@
+        //     </span>
+        // )
+
+        return null;
+    }
+
+    public static myCancerGenomeContent(myCancerGenome:string[], enable:boolean)
+    {
+        const arrowContent = <div className="rc-tooltip-arrow-inner"/>;
+        let myCancerGenomeContent:JSX.Element|null = null;
+
+        if (enable && myCancerGenome.length > 0)
+        {
+            const mcgTooltipContent = AnnotationColumnFormatter.myCancerGenomeLinks(myCancerGenome);
+
+            myCancerGenomeContent = (
+                <Tooltip overlay={mcgTooltipContent} placement="rightTop" arrowContent={arrowContent}>
+                    <span className={`${styles["annotation-item"]} mcg`}>
+                        <img width='14' height='14' src={require("./images/mcg_logo.png")} alt='My Cancer Genome Symbol' />
+                    </span>
+                </Tooltip>
+            );
+        }
+
+        return myCancerGenomeContent;
+    }
+
+    public static myCancerGenomeLinks(myCancerGenome:string[])
+    {
+        const links:any[] = [];
+
+        _.each(myCancerGenome, function(link:string){
+            links.push(
+                <li>{link}</li>
+            );
+        });
+
+        return (
+            <span>
+                <b>My Cancer Genome links:</b>
+                <br/>
+                <ul style={{"list-style-position": "inside;padding-left:0;"}}>
+                    {links}
+                </ul>
+            </span>
+        );
+    }
+
+    public static isHotspot(mutation:Mutation, map:{[key:string]: boolean}):boolean
+    {
+        let key:string = mutation.gene.hugoGeneSymbol + "_" + mutation.proteinPosStart;
+
+        if (mutation.proteinPosEnd &&
+            (mutation.proteinPosEnd !== mutation.proteinPosStart))
+        {
+            key = key + "_" + mutation.proteinPosEnd;
+        }
+
+        return map[key] || false;
+    }
+
+    public static hotspotInfo(isHotspot:boolean, is3dHotspot:boolean)
+    {
+        const title = [];
+
+        if (isHotspot)
+        {
+            title.push(<b>Recurrent Hotspot</b>);
+
+            if (is3dHotspot) {
+                title.push(" and ");
+            }
+        }
+
+        if (is3dHotspot)
+        {
+            title.push(<b>3D Clustered Hotspot</b>);
+        }
+
+
+        const publication = [];
+
+        publication.push("This mutated amino acid was identified as ");
+
+        if (isHotspot)
+        {
+            publication.push("a recurrent hotspot (statistically significant) ");
+
+            if (is3dHotspot) {
+                publication.push("and ");
+            }
+        }
+
+        if (is3dHotspot) {
+            publication.push("a 3D clustered hotspot ");
+        }
+
+        publication.push("in a population-scale cohort of tumor samples of " +
+            "various cancer types using methodology based in part on ");
+
+        if (isHotspot)
```

Comment:
@alisman updated the tooltip generating code, no more string concat or arrays. Looks a bit cleaner now.